js/index.js

```diff
@@ -16,3 +16,35 @@ for (let i = 0; i < skills.length; i++) {
     skill.innerText = skills[i];
     skillsList.appendChild(skill);
 }
+
+const messageForm = document.getElementsByName("leave_message")[0];
+
+messageForm.addEventListener("submit", messageSubmit);
+
+function messageSubmit(event) {
+    event.preventDefault()
+
+    const { name, email, message } = event.target;
+
+    console.log("name", name.value)
+    console.log("email", email.value)
+    console.log("message", message.value)
+
+    const messageSection = getElementById("messages");
+    const messageList = querySelector("ul");
+    const newMessage = document.createElement("li");
+    console.log(messageList);
+
+    newMessage.innerHTML = '<a href="mailto:' + email.value + '">';
+    newMessage.innerHTML = <span>message.value</span>;
```

Comment:
```suggestion
    newMessage.innerHTML = `<span>${message.value}<a href="mailto:${email.value}">email me!</a></span>`;
```
You're clobbering the newMessage that you set just above: You probably just want to do it all at once? When you do, make sure that you're using a string: A bare `<span>` doesn't work in plain JS. (... It does work in React, but that's a separate course. If you see examples with tags in JS, that's what's going on: It's not actually JS.)

And make sure to close the `<a>` tag.